test/reporters/base.test.ts

```diff
@@ -420,6 +420,7 @@ describe('BaseReporter', () => {
           await reporter.report(currentPriceValue, defaultReportTrigger)
           // transaction information
           expect(metricCollector!.reportTransaction).toBeCalled()
+          expect(metricCollector!.oracleBalanceValue).toBeCalled()
```

Comment:
there's no test for the actual logic of getting the balance, just that the metric function is called with something